resources/js/Pages/Collections/Components/CollectionsArticles.tsx

```diff
@@ -0,0 +1,122 @@
+import { Tab } from "@headlessui/react";
+import cn from "classnames";
+import { Fragment, useMemo, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { ArticleCard } from "@/Components/Articles/ArticleCard";
+import { ButtonLink } from "@/Components/Buttons/ButtonLink";
+import { Carousel, CarouselControls, CarouselItem } from "@/Components/Carousel";
+import { Heading } from "@/Components/Heading";
+import { Tabs } from "@/Components/Tabs";
+import { useCarousel } from "@/Hooks/useCarousel";
+
+interface Properties {
+    latest: App.Data.Articles.ArticleData[];
+    popular: App.Data.Articles.ArticleData[];
+}
+
+type TabOption = "latest" | "popular";
+
+export const CollectionsArticles = ({ latest, popular }: Properties): JSX.Element => {
+    const [tab, setTab] = useState<TabOption>("latest");
+    const articles = useMemo(() => (tab === "latest" ? latest : popular), [tab]);
+
+    const { t } = useTranslation();
+    const { slidesPerView, horizontalOffset } = useCarousel();
+
+    return (
+        <div className="articles-scroll mt-6 border-t-4 border-theme-secondary-200 pt-14 dark:border-theme-dark-800 sm:mt-8 sm:pt-8 lg:mt-12 lg:border-t-0 lg:pt-0">
+            <div className="mb-4 px-6 2xl:px-0">
+                <Heading level={1}>{t("pages.collections.articles.heading")}</Heading>
```

Comment:
https://github.com/ArdentHQ/dashbrd/pull/538 will apply the changes to the `Heading` component here once that PR is merged